src/components/flat-table/flat-table.component.js

```diff
@@ -21,6 +21,16 @@ const FlatTable = ({
   ...props
 }) => {
   const addDefaultHeight = !height && (hasStickyHead || hasStickyFooter);
+  const tableStylingProps = {
+    caption,
+    isZebra,
+    size,
+  };
+
+  if (props["aria-describedby"]) {
```

Comment:
described by is also added to a div that contains the table 
![image](https://user-images.githubusercontent.com/72866515/103287184-bf3b5680-49d9-11eb-9d60-9deb67d095d7.png) is this intentional?

Reply:
good catch, this prop was just passed with all other props. Removed.